analyzers/maven/maven.go

```diff
@@ -62,6 +63,8 @@ func New(m module.Module) (*Analyzer, error) {
 func Discover(dir string, options map[string]interface{}) ([]module.Module, error) {
 	log.WithField("dir", dir).Debug("discovering modules")
 	var modules []module.Module
+	checked := make(map[string]bool, 7)
```

Comment:
Why 7?

Reply:
prime number

Reply:
What do you mean?

Reply:
to those who are curious (Alex and I talked on Slack)... The `map` type in Go is implemented as a hash map. Although the language spec doesn't require that the "size hint" be used as the number of buckets allocated, it's common for performance reasons, reducing chances of collision, to use a prime number of buckets in a map.

Reply:
It's not the number of buckets -- it's a capacity hint; in this case, it'd actually make zero buckets 😱 . It starts making buckets at 8

Reply:
@cnr good catch if that's accurate. In any case, I agree with not introducing a speedbump to someone scanning the code for the first time—someone might wonder why it has to be 7. Remove that now :)